test/functional/test-resources.js

```diff
@@ -1058,40 +1046,6 @@ describe('Resources changeSize', () => {
       expect(overflowCallbackSpy.firstCall.args[0]).to.equal(false);
     });
 
-    it('should change size when slightly above the viewport', () => {
```

Comment:
This test actually tests element resize when below viewport. Duplicate of
https://github.com/zhouyx/amphtml/blob/3abb80f69c45c9beae069675350e10b4697069ad/test/functional/test-resources.js#L1038